project_timeline/README.rst

```diff
@@ -45,7 +45,11 @@ To view the timeline:
 * Click on the timeline view icon.
 * You will see the tasks or projects in the new view.
 
-The Task timeline uses the Start Date and End Date fields, in the Extra Info tab.
+The Task timeline uses the "Planned Start Date" and "Planned End Date" fields, in the
+"Extra Info" tab (only visible in debug mode).
+
+When a user is assigned, and there's no planned start date, current datetime is filled
```

Comment:
TBH this is working only when new task is created from kanban, but not from form: [video](https://www.loom.com/share/4d61fdfc2c7140c28b0502f617beb5a4)

@juanjosesegui-tecnativa @pedrobaeza

Reply:
That seems a glitch of v14 + the widget. If you do it in 2 steps (create it, and the assign it), it should work. It's out of the scope of the module anyway. Update to newer versions of Odoo for ORM improvements.

Reply:
Sure, it was just a heads up in case it happens in later versions